tutorials/QuantumClassification/ExploringQuantumClassificationLibrary.ipynb

```diff
@@ -69,9 +69,6 @@
     "\n",
     "# Q# configuration and necessary imports\n",
     "import qsharp\n",
-    "qsharp.packages.add(\"Microsoft.Quantum.MachineLearning::0.12.20072031\")\n",
```

Comment:
It will be loaded automatically with this PR because `Microsoft.Quantum.MachineLearning` is specified in the `.csproj` for this folder, and I've added the `IQSharpLoadAutomatically` property to that `.csproj`.

But even without this PR, the version number there wouldn't need to be specified. IQ# "knows" about that package, so it could just be:
`qsharp.packages.add("Microsoft.Quantum.MachineLearning")`
and IQ# would automatically load the package version that matches the IQ# kernel.